docs/development.md

```diff
@@ -13,12 +13,10 @@ git clone git@github.com:quantumlib/cirq.git
 cd cirq
 ```
 
-To do your development in a Docker virtual machine, you can use dev_tools/Dockerfile:
+To do your development in a Docker image, you can build one with Cirq/dev_tools/Dockerfile or pull an existing image:
```

Comment:
Cirq/dev_tools/Dockerfile doesn't exist?

Reply:
Here: https://github.com/quantumlib/Cirq/blob/master/dev_tools/Dockerfile

Reply:
@dabacon Ping :)

Reply:
Ah thanks.  Could you make the doc link to that file?

Reply:
Where should i put the link? On this file?  `docs/development.md`